stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/plugin/logic/internal/LogicRegistry.kt

```diff
@@ -122,14 +122,14 @@ internal class LogicRegistry internal constructor(
          * @param repos [RepositoryFacade] to interact with local data sources.
          * @param client An instance of [ChatClient].
          */
-        internal fun getOrCreate(
+        internal fun create(
             stateRegistry: StateRegistry,
             globalState: GlobalMutableState,
             userPresence: Boolean,
             repos: RepositoryFacade,
             client: ChatClient,
         ): LogicRegistry {
-            return instance ?: LogicRegistry(stateRegistry, globalState, userPresence, repos, client).also { logicRegistry ->
+            return LogicRegistry(stateRegistry, globalState, userPresence, repos, client).also { logicRegistry ->
                 instance = logicRegistry
```

Comment:
Can we check if `instance != null` and at least log this somewhere? Multiple instances can lead to weird behavior